File: ezSCORES/ezSCORES.Model/CompetitionsReferees.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model
{
	public class CompetitionsReferees
	{
		public int Id { get; set; }

		public int CompetitionId { get; set; }

		public int RefereeId { get; set; }
		public virtual Referees Referee { get; set; } = null!;
	}
}

File: ezSCORES/UI/ezscores_desktop/lib/models/competitionsReferees.dart
// -- competitions_referees.dart --
import 'package:ezscores_desktop/models/referees.dart';
import 'package:json_annotation/json_annotation.dart';

part 'competitionsReferees.g.dart';

@JsonSerializable()
class CompetitionsReferees {
	int? id;
	int? competitionId;
	int? refereeId;
  Referees? referee;

	CompetitionsReferees();

	factory CompetitionsReferees.fromJson(Map<String, dynamic> json) => _$CompetitionsRefereesFromJson(json);

	Map<String, dynamic> toJson() => _$CompetitionsRefereesToJson(this);
}
